cargo-shuttle/src/lib.rs

```diff
@@ -1140,8 +1140,11 @@ impl Shuttle {
         deployment_req.data = self.make_archive()?;
         if deployment_req.data.len() > CREATE_SERVICE_BODY_LIMIT {
             bail!(
-                "The project is too large - we have a {}MB project limit.",
-                CREATE_SERVICE_BODY_LIMIT / 1_000_000
+                r#"The project is too large - we have a {} MB project limit. \
+                Your project archive is {} MB. \
+                Run with `RUST_LOG="cargo_shuttle=debug"` to see which files are being packed."#,
+                CREATE_SERVICE_BODY_LIMIT / 1_000_000,
+                deployment_req.data.len() / 1_000_000,
```

Comment:
Nice, in the future, having these in a separate PR will allow for merging them faster.